ios/brave-ios/Tests/BraveWalletTests/AssetDetailStoreTests.swift

```diff
@@ -95,6 +94,26 @@ class AssetDetailStoreTests: XCTestCase {
       $1(true)
     }
 
+    let mockBtcBalance: Double = 0.001
+    let btcBalanceSatoshi =
+      formatter.weiString(
+        from: mockBtcBalance,
+        radix: .decimal,
+        decimals: Int(BraveWallet.BlockchainToken.mockBTCToken.decimals)
+      ) ?? ""
+    let bitcoinWalletService = BraveWallet.TestBitcoinWalletService()
+    bitcoinWalletService._balance = {
+      $1(
+        .init(
+          totalBalance: UInt64(btcBalanceSatoshi) ?? 0,
+          availableBalance: UInt64(btcBalanceSatoshi) ?? 0,
+          pendingBalance: 0,
+          balances: [:]
+        ),
+        nil
+      )
+    }
```

Comment:
nit: This unit test is for `previewToken` / ETH on Ethereum Mainnet. Shouldn't need to return any BTC balance here.